client/netmap_test.go

```diff
@@ -145,6 +145,6 @@ func TestClient_NetMapSnapshot(t *testing.T) {
 
 	res, err = c.NetMapSnapshot(ctx, prm)
 	require.NoError(t, err)
-	require.True(t, apistatus.IsSuccessful(res.Status()))
+	require.True(t, apistatus.IsSuccessful(res))
```

Comment:
This line should just be dropped, I think, `IsSuccessful` will never fail for `res`.

Reply:
removed